README.md

```diff
@@ -19,15 +19,15 @@ This library builds on top of the [Microsoft.OpenAPI.NET](https://github.com/mic
 
-| Language | Generation | Abstractions                   | Serialization | Authentication | HTTP | Required tools & dependencies |
-| -------- | ---------- |--------------------------------| ------------- | -------------- | ---- | -------------- |
-| CSharp | [✔](https://github.com/microsoft/kiota/projects/5) | [✔](./abstractions/dotnet)     | [JSON](./serialization/dotnet/json) | [Anonymous](./abstractions/dotnet/src/authentication/AnonymousAuthenticationProvider.cs), [Azure](./authentication/dotnet/azure) | [✔](./http/dotnet/httpclient) | [link](https://microsoft.github.io/kiota/get-started/dotnet) |
-| Go | [✔](https://github.com/microsoft/kiota/projects/8) | [✔](./abstractions/go)         | [JSON](./serialization/go/json) | [Anonymous](./abstractions/go/authentication/anonymous_authentication_provider.go), [Azure](./authentication/go/azure) | [✔](./http/go/nethttp) | [link](https://microsoft.github.io/kiota/get-started/go) |
-| Java | [✔](https://github.com/microsoft/kiota/projects/7) | [✔](./abstractions/java)       | [JSON](./serialization/java/json) | [Anonymous](./abstractions/java/lib/src/main/java/com/microsoft/kiota/authentication/AnonymousAuthenticationProvider.java), [Azure](./authentication/java/azure) | [✔](./http/java/okhttp) | [link](https://microsoft.github.io/kiota/get-started/java) |
-| PHP | [✔](https://github.com/microsoft/kiota/projects/4) | [✔](./abstractions/php)          | ❌ | ❌ | ❌ |  |
-| Python | [❌](https://github.com/microsoft/kiota/projects/3) | ❌                              | ❌ | ❌ | ❌ |  |
-| Ruby | [✔](https://github.com/microsoft/kiota/projects/6) | [✔](./abstractions/ruby)       | [JSON](./serialization/ruby/json/microsoft_kiota_serialization) | [Anonymous](./abstractions/ruby/microsoft_kiota_abstractions/lib/microsoft_kiota_abstractions/authentication/anonymous_authentication_provider.rb), [❌ Azure](https://github.com/microsoft/kiota/issues/421) | [✔](./http/ruby/nethttp/microsoft_kiota_nethttplibrary)| [link](https://microsoft.github.io/kiota/get-started/ruby)  |
-| TypeScript/JavaScript | [✔](https://github.com/microsoft/kiota/projects/2) | [✔](./abstractions/typescript) | [JSON](./serialization/typescript/json) | [Anonymous](./abstractions/typescript/src/authentication/anonymousAuthenticationProvider.ts), [Azure](./authentication/typescript/azure) | [✔](./http/typescript/fetch) | [link](https://microsoft.github.io/kiota/get-started/typescript) |
+| Language | Generation | Abstractions | Serialization | Authentication | HTTP | Required tools & dependencies |
+| -------- | ---------- | ------------ | ------------- | -------------- | ---- | -------------- |
+| CSharp | [✔](https://github.com/microsoft/kiota/projects/5) | [✔](./abstractions/dotnet) | [JSON](./serialization/dotnet/json) | [Anonymous](./abstractions/dotnet/src/authentication/AnonymousAuthenticationProvider.cs), [Azure](./authentication/dotnet/azure) | [✔](./http/dotnet/httpclient) | [link](./docs/requiredtools/dotnet.md) |
```

Comment:
Please revert that change and put back the links to the github.io site when available (dotnet, java, go, ruby, typescript)

Reply:
but just to be clear, this page still needs to be updated for the Python row